src/components/UserAccess/UserAccessListView.tsx

```diff
@@ -80,6 +82,11 @@ export const UserAccessListView: React.FC<React.PropsWithChildren<unknown>> = ()
       ),
     [roleBindings, usernameFilter],
   );
+
+  if (error) {
+    return errorState;
+  }
```

Comment:
you can basically write this as 
```suggestion
  if (error) {
    return useErrorState(error, loaded, 'role bindings');
  }
```